### ISSUE TITLE
Possible Plugins Issue with Multiple Buffers/Tabs

### ISSUE THREAD

**Issue by Tientuine** [begriffs/haskell-vim-now#230]
First, let me thank begriffs for his patience and helpfulness with my previous Issue. To be honest, I overlooked some of the very clearly written README customization section, and many other folks might have (justifiably) chastised me for that. I'm getting used to the haskell-vim-now environment, and enjoying it so far. Now on to my current Issue...

When opening multiple files, whether in regular buffers or in Vim tabs, I notice that some plugins appear to behave differently for the very first buffer than for all remaining/subsequent buffers. In particular, the syntax highlighting differs, and some of the Unicode concealing differs. I have searched around but can't seem to find an explanation of why this might be happening. I have tried toggling syntax highlighting and even running PlugInstall again, but to no avail. Here are screenshots of my situation:

What the first buffer/tab looks like:
![compare1](https://cloud.githubusercontent.com/assets/1669148/22917040/6abcb3c2-f250-11e6-84d3-a42d7fabdef7.png)

What all other buffers/tabs look like:
![compare2](https://cloud.githubusercontent.com/assets/1669148/22917043/72cad7f6-f250-11e6-8204-0960954b3d05.png)

Not that some highlighting and concealing is the same, but not all. The only changes I have made from the default HVN setup include (a) solarized color scheme, (b) solarized theme for airline, and (c) a few modifications to the haskell.vim conceal script. However, to rule out my changes as the cause, I also reverted back to vanilla HVN and still notice the same behavior.

I'm hoping someone else has experienced this also and/or has a good guess about a possible cause. Thanks!

**Comment by begriffs** [comment]
Do you see a similar problem between splits in a single window as between tabs?

**Comment by Tientuine** [comment]
Yes, the problem is present in split window. When I open two files, for example `vim -o AST.hs Interp.hs`, the first one has the same styling as if I opened that file alone while the second appears to be missing some colors and unicode concealings.

I don't know it it is in any way related, but in the first file (and when opening a single file) I often see the a strange character sequence at the end of the last visible line, something like `^[[>1;4601;0c`. It seems that this might be some rendering issue with of end-of-line, as it tends to disappear if I edit the file (especially nearby text). Interestingly, this does not appear in the second/subsequent window/buffer.

**Comment by begriffs** [comment]
I'm not sure what this is to be honest. My installation works fine. :(
